src/backer/backering.py

```diff
@@ -299,34 +300,37 @@ def on_post(self, req, rep):
         cr = httping.parseCesrHttpRequest(req=req)
         serder = serdering.SerderKERI(sad=cr.payload, kind=eventing.Serials.json)
 
-        backer_identifiers = serder.ked["b"]
-
-        # Confirm registry backer
-        # raise exception when invalid registry backer or invalid identifier
-        if TraitCodex.Backers not in serder.ked["c"] or self.hab.pre not in backer_identifiers:
-            raise falcon.HTTPBadRequest(falcon.HTTP_400)
+        ilk = serder.ked["t"]
+        msg = bytearray(serder.raw)
+        msg.extend(cr.attachments.encode("utf-8"))
 
-        valid_backer_seals = 0
+        # Should check when create identifiers only
+        if ilk in (Ilks.icp, Ilks.rot):
```

Comment:
Can you change the comment to: `@TODO - Should allow other methods, and check that backer trait set most recently to Cardano`, as the current comment is wrong.